contracts/Launchpeg.sol

```diff
@@ -162,6 +173,7 @@ contract Launchpeg is BaseLaunchpeg, ILaunchpeg {
         uint256 _allowlistStartTime,
         uint256 _allowlistDiscountPercent,
         uint256 _publicSaleStartTime,
+        uint256 _publicSaleEndTime,
         uint256 _publicSaleDiscountPercent
     ) external override onlyOwner atPhase(Phase.NotStarted) {
         if (_auctionSaleStartTime < block.timestamp) {
```

Comment:
@Mathieu-Be i noticed that the launchpeg timestamps are such that

`block.timestamp` <= `auctionSaleStartTime` < `allowlistStartTime` <= `publicSaleStartTime` ...

and flat launchpeg timestamps are

`block.timestamp` <= `allowlistStartTime` <= `publicSaleStartTime` ...

is there a reason that auction sale time cannot be equal to allowlist start time, but we allow equality for other phases? probably not a huge deal - just curious

Reply:
We put `allowlistStartTime = publicSaleStartTime` when there is no allowlist phase, but putting `auctionSaleStartTime = allowlistStartTime` and skipping the dutch auction phase doesn't make much sense, you should better use FlatLaunchpeg in that case